### ISSUE TITLE
Podmins need the ability to ban users BY IP ADDRESS.

### ISSUE THREAD

**Issue by Tectract** [diaspora/diaspora#6517]
Sadly :(

Would this be a difficult PR?

**Comment by Tectract** [comment]
The pod I administer does not actually log IP addresses by default, and they are completely scrubbed from all server logs also. So one thing that would be super-nice is the ability to turn on/off IP tracking by default, and by username. I could maybe offer a small bitcoin bounty :D

**Comment by jhass** [comment]
Maybe there's a devise plugin we can use. Somebody would need to do some research on this.

**Comment by jhass** [comment]
@Tectract in the mean time you should be able to ban by IP at the reverse proxy or packet filter (iptables) level, just blackhole all traffic coming from it.

**Comment by Tectract** [comment]
Yes, banning an IP from access to my server is not terribly difficult once I identify the IP I want to ban, but there is not easy way to do it seems. I'd have to go digging through the database and the server logs to even begin to try to figure out what IP the offending user is connecting from.
